test/Passes/DefaultPipeline/linalg-memref.mlir

```diff
@@ -174,66 +150,6 @@ func.func @relu_mapping_fail(%arg0: memref<10x10xf32>, %arg1: memref<10x10xf32>)
 
 // -----
 
-#map = affine_map<(d0, d1, d2, d3) -> (d3)>
-#map1 = affine_map<(d0, d1, d2, d3) -> (d0, d1, d2, d3)>
-
-// CHECK-LABEL: @identity_mapping
-func.func @identity_mapping(%arg0: memref<64xf32>) -> memref<12x56x56x64xf32> {
-  // CHECK: call @xsmm_unary_dispatch
-  // CHECK: scf.parallel
-  // CHECK:   %[[cast:.*]] = memref.cast
-  // CHECK:   %[[cast1:.*]] = memref.cast
-  // CHECK:   call @xsmm_unary_invoke({{.*}}%[[cast]], %[[cast1]]
-  %alloc = memref.alloc() {alignment = 128 : i64} : memref<12x56x56x64xf32>
-  linalg.generic {indexing_maps = [#map, #map1], iterator_types = ["parallel", "parallel", "parallel", "parallel"], library_call="tpp.identity" } ins(%arg0 : memref<64xf32>) outs(%alloc : memref<12x56x56x64xf32>) {
-    ^bb0(%in: f32, %out: f32):
-      linalg.yield %in : f32
-  }
-
-  return %alloc : memref<12x56x56x64xf32>
-}
-
-// -----
-
-#map = affine_map<(d0, d1, d2) -> (d0, d1, d2)>
-func.func @add_mapping(%arg0: memref<1x10x10xf32>, %arg1: memref<1x10x10xf32>) {
```

Comment:
We could keep at least this one, CHECK-NOT tpp, with a comment that we don't auto-tile 3D left-overs.